src/lib/TwaManifest.ts

```diff
@@ -120,6 +120,7 @@ export class TwaManifest {
   signingKey: SigningKeyInfo;
   appVersion: string;
   shortcuts: string;
+  generatorApp: string;
```

Comment:
On the subject of constants, can any of these members be constant?

Reply:
Not the members. But moving sone of the "magic values" throughout the code to consts sound like a good idea.

Reply:
Moved the default values to consts. Not sure what the best practice for JSON field names is.